gui/masspit.lua

```diff
@@ -0,0 +1,218 @@
+-- Interface to assign caged units in a stockpile to pond/pit.
+
+local utils = require 'utils'
+local gui = require('gui')
+local widgets = require('gui.widgets')
+
+Masspit = defclass(Masspit, widgets.Window)
+Masspit.ATTRS {
+    frame_title='masspit',
```

Comment:
This is what players actually see on the window, so a more "title-like" phrasing would be more appropriate. How about "Mass-pit caged creatures". Just "Mass-pit" might be ok, but it's kind of a strange terminology that not everyone might understand.